src/Integration.UnitTests/MefServices/ActiveSolutionBoundTrackerTests.cs

```diff
@@ -349,8 +349,48 @@ public void OnBindingStateChanged_NewConfiguration_EventsRaisedInCorrectOrder()
                 // Assert
                 // Different config so event should be raised
                 eventCounter.PreSolutionBindingChangedCount.Should().Be(1);
+                eventCounter.PreSolutionBindingUpdatedCount.Should().Be(0);
+                eventCounter.SolutionBindingChangedCount.Should().Be(1);
                 eventCounter.SolutionBindingUpdatedCount.Should().Be(0);
+
+                eventCounter.RaisedEventNames.Should().HaveCount(2);
+                eventCounter.RaisedEventNames[0].Should().Be("PreSolutionBindingChanged");
```

Comment:
Use name of instead of hardcoded string for event name.
Maybe search and replace in the whole file?

Reply:
I didn't change these, but I can try to fix this.